src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -255,8 +322,11 @@ export default class GeneBarPlot extends React.Component<
                                         this.selectedGenes = genes;
                                         this._label = label;
                                         this.isGeneSelectionPopupVisible = false;
+                                        if (this.props.comparisonStore)
+                                            this.props.comparisonStore.alterationAndPathwaysFlag = value;
```

Comment:
Not sure if we need a component specific flag like this one. Ideally, this should be more generic.